src/Domain/CoordinateMaps/Frustum.hpp

```diff
@@ -21,27 +21,238 @@ class er;
 namespace domain {
 namespace CoordinateMaps {
 
-/// \ingroup CoordinateMapsGroup
-///
-/// Frustum map from the cube to a rectangular frustum where the bases of the
-/// frustum are perpendicular to the z-axis. The `lower_bound` and
-/// `upper_bound` values correspond to the locations of the bases along this
-/// axis. The `face_vertices` values determine the size of the two rectangular
-/// bases along the other two axes. For example, for a frustum along the z-axis,
-/// with the lower base starting at (x,y) = (-2.0,3.0) and extending to
-/// (2.0,5.0), and with the upper base extending from (0.0,1.0) to (1.0,3.0),
-/// the corresponding value for `face_vertices` is `{{{{-2.0,3.0}}, {{2.0,5.0}},
-/// {{0.0,1.0}}, {{1.0,3.0}}}}`. The user may reorient the frustum by passing
-/// an `OrientationMap` to the constructor. If `with_equiangular_map` is true,
-/// then this coordinate map applies a tangent function mapping to the logical
-/// xi and eta coordinates.
+/*!
+ * \ingroup CoordinateMapsGroup
+ *
+ * \brief A reorientable map from the cube to a frustum.
+ * \image html frustum_table1.png "A frustum with rectangular bases."
+ *
+ * A map from the cube to the volume determined by interpolating between two
+ * parallel rectangles each perpendicular to the \f$z\f$-axis. A Frustum map
+ * \f$\vec{x}(\xi,\eta,\zeta)\f$ is determined by specifying two
+ * heights \f$z_1 = \f$ `lower_bound` and \f$z_2 = \f$ `upper_bound` for the
+ * positions of the rectangular bases of the frustum along the \f$z-\f$axis,
+ * and the sizes of the rectangles are determined by the eight values passed to
+ * `face_vertices`:
+ *
+ * \f{align*}{
+ * &\textrm{lower x upper base} : x^{+\zeta}_{-\xi,-\eta} = x(-1,-1,1)\\
```

Comment:
These variable names do not refer to individual corners of the frustum, but rather to the extents along each dimension, since the bases are rectangular. In other words, the value `lower_x_upper_base` is equal to not only x(-1,-1,1) but also x(-1,1,1) (indeed it is equal to x(-1,eta,1) for all eta, maybe I write that instead?)